examples/assets-and-custom-transfer-logic/README.md

```diff
@@ -0,0 +1,42 @@
+# Multisignature delegated account support
```

Comment:
Shouldn't we change the title?

Reply:
Also let's rename the directory: 
`assets-and-custom-transfer-logic` -> `restricted-assets`

Reply:
Thanks for catching the title change. Updating directory name as well.